docs/part3.md

```diff
@@ -1 +1,178 @@
 # Part III: Numerical precision in _ab initio_ calculations.
+
+In this part you will learn about several new quantities and tools for working with numerical convergence in NOMAD.
+These tools will help you
+
+1. query for the level of precision or conformity that you need.
+2. deploy extracted data into a notebook to start performing data science.
+
+Your starting point is the Entries page > FILTERS (side menu) > Precision.
+To navigate to the Entries page, check out [Part I - Exploring NOMAD](part1.md#entries_section).
+
+<!-- Screenshot of the Precision section -->
+
+Since these quantities are new and might not be commonly known, the emphasis is placed here on their definitions, as well as, the bare minimum of theoretical knowledge required to handle them.
+There are also a couple example instructions guiding you to a specific entry or for downloading processed data into a notebook.
+Also watch out for the boxes with a pencil sign.
+They delve deeper into some topics that are too tangential for the tutorial.
+
+## Reciprocal space
+
+In periodic systems, the most universal numerical parameter is the integration of the reciprocal, or k-space, and its sampling.
+With the sampling points often being spaced at fixed intervals, one can define a homogeneous _k-density_
+$= \frac{\text{no. k-points}}{||\text{k-lattice vector}||}$.
+As the k-density ramps up, the Bloch wavefunction converges. <!-- phrase better -->
+Note that each dimension has its own k-density, which do not have to coincide.
+To ensure that NOMAD users obtain data that meets their convergence needs, **k-line density** only shows the lowest value.
+
+<p align="center">
+    <img src="../assets/part3_convergence/klinedensity.png" alt="Schematic representation of k-line density calculation." width="65%" title="k-line density for 3D and 2D lattices">
+</p>
```

Comment:
I think this image requires some explanation, if you want to include it here. Probably also adding a screenshot of the filter in NOMAD.

Reply:
Well, the explanation is the text above (and the hoover text a little bit).
There is now a screenshot a bit above, yes. Is it clearer like that?